packages/block-editor/src/components/block-list/index.js

```diff
@@ -48,21 +48,23 @@ const delayedBlockVisibilityDebounceOptions = {
 };
 
 function Root( { className, ...settings } ) {
-	const { isOutlineMode, isFocusMode, temporarilyEditingAsBlocks } =
-		useSelect( ( select ) => {
+	const { isOutlineMode, isFocusMode, editedContentOnlySection } = useSelect(
+		( select ) => {
 			const {
 				getSettings,
-				getTemporarilyEditingAsBlocks,
+				getEditedContentOnlySection,
 				isTyping,
 				hasBlockSpotlight,
 			} = unlock( select( blockEditorStore ) );
 			const { outlineMode, focusMode } = getSettings();
 			return {
 				isOutlineMode: outlineMode && ! isTyping(),
 				isFocusMode: focusMode || hasBlockSpotlight(),
-				temporarilyEditingAsBlocks: getTemporarilyEditingAsBlocks(),
+				editedContentOnlySection: getEditedContentOnlySection(),
```

Comment:
Should this be `editableContentOnlySection` instead? The selector is looking at whether it's in an editable state or not, isn't it?